DueTimer.cpp

```diff
@@ -160,7 +169,7 @@ uint8_t DueTimer::bestClock(double frequency, uint32_t& retRC){
 	float bestError = 9.999e99;
 	do
 	{
-		ticks = (float) VARIANT_MCK / frequency / (float) clockConfig[clkId].divisor;
+		ticks = (float) SystemCoreClock / frequency / (float) clockConfig[clkId].divisor;
```

Comment:
VARIANT_MCK is a constant defined at compile time, SystemCoreClock is a run-time variable common to all Cortex-M implementations that use CMSIS libraries. The use of a constant prevents correct timer setup if your application code wants to change the system clock to enter low-power modes. I do not see any problems for Arduino Due: if you never change the clock at run-time, SystemCoreClock has the same value as VARIANT_MCK.
Using SystemCoreClock allows you to call stop()/start() to reconfigure the timers to have the same frequency, even after the new system clock is applied. If you keep using the constant VARIANT_MCK, changes to the system clock will affect the timer frequency.